demo/app.js

```diff
@@ -60,7 +60,7 @@ class App extends React.Component {
                   : 'none'
               }}
             >
-              Nuka Carousel: Slide {currentSlide + 1}
+              Nuka Carousel: Slide {Math.ceil(currentSlide) + 1}
```

Comment:
Depending how the slidesToShow is set, sometimes the current slide can come back as a fraction (not sure if this is a bug or a feature), so I'm rounding it to `ceil` so they always appear as a solid int for testing purposes.